config.json

```diff
@@ -868,8 +868,8 @@
       {
         "groupId": "com.google.firebase",
         "artifactId": "firebase-auth-ktx",
-        "version": "21.0.8",
-        "nugetVersion": "121.0.8",
+        "version": "21.0.0",
```

Comment:
This version went backwards. Looks like it should be `21.1.0`?

Reply:
yup
typo

https://www.nuget.org/packages/Xamarin.Firebase.Auth.Ktx/121.0.8

https://maven.google.com/web/index.html?q=firebase-auth-ktx#com.google.firebase:firebase-auth-ktx:21.1.0